lib/LedgerEventStorage.js

```diff
@@ -142,6 +142,37 @@ class LedgerEventStorage {
     return {dupHashes};
   }
 
+  /**
+   * Filter a list of eventHashes.
+   *
+   * @param eventHash - a string or array of hashes to filter.
+   * @param blockHeight - filter on `meta.blockHeight`.
+   * @param consensus - filter on `meta.consensus`.
+   *
+   * @return a Promise that resolves to the hashes that satisfy the filter.
+   */
+  // only blockHeight OR consensus filter is allowed
+  // this is because a combined query cannot be covered under an index
+  // see audit below for details
```

Comment:
Having these comments here may break the doc parser ... move into the function's docs section above as notes or into the code?

Reply:
Done.